### ISSUE TITLE
Ensure that service outages for oawaiver.princeton.edu trigger DataDog alerts for members of RDSS

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/oawaiver#71]
This is in response to a service outage which was reported by users at 09:46 EDT this morning. This is to ensure that DataDog alerts are configured for oawaiver.princeton.edu over port 443.

**Comment by jrgriffiniii** [comment]
The following errors were logged while trying to install the DataDog client on the server:

```bash
TASK [roles/datadog : Ensure Datadog repository is up-to-date] ***********************************************************************
fatal: [oawaiver-staging1.princeton.edu]: FAILED! => {"changed": false, "msg": "Failed to update apt cache: W:GPG error: https://apt.datadoghq.com stable Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 33EE313BAD9589B7, E:The repository 'https://apt.datadoghq.com stable Release' is not signed., W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., W:Skipping acquire of configured file 'Release/binary-amd64/Packages' as repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic InRelease' doesn't have the component 'Release' (component misspelt in sources.list?), W:Skipping acquire of configured file 'Release/binary-i386/Packages' as repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic InRelease' doesn't have the component 'Release' (component misspelt in sources.list?), W:Skipping acquire of configured file 'Release/i18n/Translation-en_US' as repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic InRelease' doesn't have the component 'Release' (component misspelt in sources.list?), W:Skipping acquire of configured file 'Release/i18n/Translation-en' as repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic InRelease' doesn't have the component 'Release' (component misspelt in sources.list?)"}
```

**Comment by jrgriffiniii** [comment]
This seems to be the root cause of the problem:

```bash
W: GPG error: https://apt.datadoghq.com stable Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 33EE313BAD9589B7
```
